src/components/2FA/types.ts

```diff
@@ -0,0 +1,8 @@
+import { AuthResponse, FunctionWithParams } from 'types'
+
+export interface User2FAComponentProps {
+  userId: string
+  handleCloseModal: VoidFunction
+  onSuccessCallback: FunctionWithParams<AuthResponse>
+  onErrorCallback?: FunctionWithParams<string>
+}
```

Comment:
Props should be near the component (in mostly cases upper the component declaration). Please move it in component file

Reply:
Corrected